recipes-ni/ni-systemreplication/ni-systemreplication.bb

```diff
@@ -0,0 +1,25 @@
+SUMMARY = "A system replication utility for NI LinuxRT"
+DESCRIPTION = "Installs the nisystemreplication utility"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SECTION = "base"
+
+SRC_URI = "\
+    file://nisystemreplication \
+"
+
+S = "${WORKDIR}"
+
+do_install () {
+    install -d ${D}${bindir}
+    install -m 0550 ${S}/nisystemreplication ${D}${bindir}
```

Comment:
Oh. Actually: let's put this in `/sbin`. It doesn't make sense for anyone but root to run this tool; right?

Reply:
Done. Yes, only root is supposed to run.